opsdroid/testing/external_api.py

```diff
@@ -122,14 +122,22 @@ def base_url(self) -> str:
         return f"http://{self.host}:{self.port}"
 
     def add_response(
-        self, route: str, method: str, response_path: str, status: int = 200
+        self,
+        route: str,
+        method: str,
+        response: Union[dict, PathLike] = None,
```

Comment:
I figured you might not always want to load it from a file, so just supporting a dict here would be good?

Reply:
:+1: